tailf_test.go

```diff
@@ -28,52 +38,69 @@ func canFollowFile(t *testing.T, filename string, file *os.File) error {
 
 	follow, err := tailf.Follow(filename, true)
 	if err != nil {
-		return fmt.Errorf("creating follower: %v", err)
+		t.Fatalf("Failed creating tailf.follower: '%v'", err)
```

Comment:
invoking `t.Fatalf` won't run the `defer`s, so the temp file created for test will be left on disk.

also use uncapitalized messages to fit with the rest of the tests.